Fix a test failure with recent versions of ... Pytest?  Python?

Taken from upstream:
https://github.com/keras-team/keras/commit/fe35050a8f18dc52304aa8da4e463eececa25240

diff --git a/tests/integration_tests/applications_test.py b/tests/integration_tests/applications_test.py
index 979f2f2abd6..6e3b57fa8e2 100644
--- a/tests/integration_tests/applications_test.py
+++ b/tests/integration_tests/applications_test.py
@@ -58,7 +58,8 @@ def _test_application_basic(app, last_dim=1000):
 def _test_application_notop(app, last_dim):
     output_shape = _get_output_shape(
         lambda: app(weights=None, include_top=False))
-    assert output_shape == (None, None, None, last_dim)
+    assert len(output_shape) == 4
+    assert output_shape[-1] == last_dim
 
 
 def test_mobilenet_v2_legacy_import():
